### PR TITLE
PEP8 style checking for python bindings

### PR DESCRIPTION
**Pull request by qrayven**
# Description of change

Add linter autpep8 check to the CI

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Enhancement (a non-breaking change which adds functionality)

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: bindings/python/tox.ini [hunk 1]
```diff
@@ -15,4 +15,14 @@ passenv = *
...
+
+[testenv:lint]
+description = Run linter {basepython}
+deps = autopep8
+commands =
+    autopep8 --exclude .venv  --diff --recursive --aggressive --exit-code .
+
+
+
+
```
Comment (Thoralf-M) [inline bindings/python/tox.ini]:
```suggestion
```

File: bindings/python/examples/exchange/1_create_account.py [hunk 2]
```diff
@@ -30,7 +31,8 @@
 wallet = Wallet(os.environ.get('WALLET_DB_PATH'),
                 client_options, CoinType.IOTA, secret_manager)
 
-# Store the mnemonic in the Stronghold snapshot, this only needs to be done once
+# Store the mnemonic in the Stronghold snapshot, this only needs to be
+# done once
```
Comment (thibault-martinez) [inline bindings/python/examples/exchange/1_create_account.py]:
```suggestion
# done once.
```

File: bindings/python/examples/client/submit_and_read_block.py [hunk 3]
```diff
@@ -90,4 +90,5 @@
 print(f'  {message_out}')
 
 # Or see the message online, with the testnet explorer
```
Comment (thibault-martinez) [inline bindings/python/examples/client/submit_and_read_block.py]:
```suggestion
# Or see the message online, with the testnet explorer.
```

File: bindings/python/examples/client/05_get_address_balance.py [hunk 4]
```diff
@@ -17,7 +17,8 @@
     has_storage_deposit_return=False
 )
 
-# Get output ids of basic outputs that can be controlled by this address without further unlock constraints
+# Get output ids of basic outputs that can be controlled by this address
+# without further unlock constraints
```
Comment (thibault-martinez) [inline bindings/python/examples/client/05_get_address_balance.py]:
```suggestion
# without further unlock constraints.
```

File: bindings/python/examples/how_tos/accounts_and_addresses/consolidate_outputs.py [hunk 5]
```diff
@@ -16,7 +17,8 @@
 
 account = wallet.get_account('Alice')
 
-# Sync account to make sure account is updated with outputs from previous examples
+# Sync account to make sure account is updated with outputs from previous
+# examples
```
Comment (thibault-martinez) [inline bindings/python/examples/how_tos/accounts_and_addresses/consolidate_outputs.py]:
```suggestion
# examples.
```

File: bindings/python/examples/how_tos/accounts_and_addresses/consolidate_outputs.py [hunk 6]
```diff
@@ -41,7 +43,8 @@
 print('Sending consolidation transaction...')
 
 # Consolidate unspent outputs and print the consolidation transaction ID
-# Set `force` to true to force the consolidation even though the `output_consolidation_threshold` isn't reached
+# Set `force` to true to force the consolidation even though the
+# `output_consolidation_threshold` isn't reached
```
Comment (thibault-martinez) [inline bindings/python/examples/how_tos/accounts_and_addresses/consolidate_outputs.py]:
```suggestion
# `output_consolidation_threshold` isn't reached.
```

File: bindings/python/examples/how_tos/advanced_transactions/claim_transaction.py [hunk 7]
```diff
@@ -4,7 +4,8 @@
 
 load_dotenv()
 
-# In this example we will claim outputs that have additional unlock conditions as expiration or storage deposit return
+# In this example we will claim outputs that have additional unlock
+# conditions as expiration or storage deposit return
```
Comment (thibault-martinez) [inline bindings/python/examples/how_tos/advanced_transactions/claim_transaction.py]:
```suggestion
# conditions as expiration or storage deposit return.
```

File: bindings/python/examples/how_tos/accounts_and_addresses/create_account.py [hunk 8]
```diff
@@ -18,12 +18,17 @@
...
+    client_options,
+    coin_type,
+    secret_manager)
 
 if 'MNEMONIC' not in os.environ:
     raise Exception(".env MNEMONIC is undefined, see .env.example")
 
-# Store the mnemonic in the Stronghold snapshot, this only needs to be done once
+# Store the mnemonic in the Stronghold snapshot, this only needs to be
+# done once
```
Comment (thibault-martinez) [inline bindings/python/examples/how_tos/accounts_and_addresses/create_account.py]:
```suggestion
# done once.
```

File: bindings/python/examples/wallet/12-prepare_output.py [hunk 9]
```diff
@@ -5,7 +5,8 @@
 
 load_dotenv()
 
-# In this example we will prepare an output with an address and expiration unlock condition and send it
+# In this example we will prepare an output with an address and expiration
+# unlock condition and send it
```
Comment (thibault-martinez) [inline bindings/python/examples/wallet/12-prepare_output.py]:
```suggestion
# unlock condition and send it.
```

File: bindings/python/iota_sdk/client/client.py [hunk 10]
```diff
@@ -88,7 +88,8 @@ def __init__(
         """
         client_config = dict(locals())
         del client_config['self']
-        # Delete client_handle, because it's not needed here and can't be serialized
+        # Delete client_handle, because it's not needed here and can't be
+        # serialized
```
Comment (thibault-martinez) [inline bindings/python/iota_sdk/client/client.py]:
```suggestion
        # serialized.
```

File: bindings/python/iota_sdk/types/common.py [hunk 11]
```diff
@@ -7,7 +7,8 @@
 HexStr = NewType("HexStr", str)
 
 HD_WALLET_TYPE = 44
-HARDEN_MASK = 1 << 31;
+HARDEN_MASK = 1 << 31
```
Comment (thibault-martinez) [inline bindings/python/iota_sdk/types/common.py]: Are these still used since we have Bip44?

File: bindings/python/tox.ini (hunk 1 above)
Comment (thibault-martinez) [inline bindings/python/tox.ini]: Why so many lines?

File: bindings/python/iota_sdk/types/common.py (hunk 11 above)
Comment (Thoralf-M) [reply bindings/python/iota_sdk/types/common.py]: No

File: bindings/python/examples/how_tos/client/get_outputs.py [hunk 12]
```diff
@@ -17,7 +17,8 @@
     has_storage_deposit_return=False
 )
 
-# Get output ids of basic outputs that can be controlled by this address without further unlock constraints
+# Get output ids of basic outputs that can be controlled by this address
+# without further unlock constraints
```
Comment (thibault-martinez) [inline bindings/python/examples/how_tos/client/get_outputs.py]:
```suggestion
# without further unlock constraints.
```

File: bindings/python/iota_sdk/types/common.py (hunk 11 above)
Comment (thibault-martinez) [reply bindings/python/iota_sdk/types/common.py]: @qrayven Please remove

File: bindings/python/examples/how_tos/native_tokens/create.py [hunk 13]
```diff
@@ -18,7 +18,9 @@
 # Sync account with the node
 balance = account.sync()
 
-# We can first check if we already have an alias in our account, because an alias can have many foundry outputs and therefore we can reuse an existing one
+# We can first check if we already have an alias in our account, because
+# an alias can have many foundry outputs and therefore we can reuse an
+# existing one
```
Comment (thibault-martinez) [inline bindings/python/examples/how_tos/native_tokens/create.py]:
```suggestion
# existing one.
```

File: bindings/python/examples/wallet/backup.py [hunk 14]
```diff
@@ -19,12 +19,13 @@
...
 wallet = Wallet('./backup-database', client_options,
-                    coin_type, secret_manager)
+                coin_type, secret_manager)
 
 if 'MNEMONIC' not in os.environ:
     raise Exception(".env MNEMONIC is undefined, see .env.example")
 
-# Store the mnemonic in the Stronghold snapshot, this only needs to be done once
+# Store the mnemonic in the Stronghold snapshot, this only needs to be
+# done once
```
Comment (thibault-martinez) [inline bindings/python/examples/wallet/backup.py]:
```suggestion
# done once.
```

File: bindings/python/examples/wallet/logger.py [hunk 15]
```diff
@@ -25,15 +26,18 @@
...
 wallet = Wallet(os.environ['WALLET_DB_PATH'], client_options,
-                    coin_type, secret_manager)
+                coin_type, secret_manager)
 
 if 'MNEMONIC' not in os.environ:
     raise Exception(".env MNEMONIC is undefined, see .env.example")
 
-# Store the mnemonic in the Stronghold snapshot, this only needs to be done once
+# Store the mnemonic in the Stronghold snapshot, this only needs to be
+# done once
```
Comment (thibault-martinez) [inline bindings/python/examples/wallet/logger.py]:
```suggestion
# done once.
```

File: bindings/python/examples/wallet/recover_accounts.py [hunk 16]
```diff
@@ -19,15 +19,21 @@
...
+    client_options,
+    coin_type,
+    secret_manager)
 
 if 'MNEMONIC' not in os.environ:
     raise Exception(".env MNEMONIC is undefined, see .env.example")
 
-# Store the mnemonic in the Stronghold snapshot, this only needs to be done once
+# Store the mnemonic in the Stronghold snapshot, this only needs to be
+# done once
```
Comment (thibault-martinez) [inline bindings/python/examples/wallet/recover_accounts.py]:
```suggestion
# done once.
```

File: bindings/python/examples/wallet/recover_accounts.py [hunk 17]
```diff
@@ -19,15 +19,21 @@
...
 account = wallet.store_mnemonic(os.environ['MNEMONIC'])
 
 # Searches for unspent outputs until no ones are found for 3 accounts in a row
-# and checks the addresses for each account until 10 addresses in a row have nothing
+# and checks the addresses for each account until 10 addresses in a row
+# have nothing
```
Comment (thibault-martinez) [inline bindings/python/examples/wallet/recover_accounts.py]:
```suggestion
# have nothing.
```

File: bindings/python/tox.ini [hunk 18]
```diff
@@ -15,4 +15,14 @@ passenv = *
...
     pip install .
-    pytest
\ No newline at end of file
+    pytest
+
+[testenv:lint]
+description = Run linter {basepython}
+deps = autopep8
+commands =
+    autopep8 --exclude .venv  --diff --recursive --aggressive --exit-code .
```
Comment (thibault-martinez) [inline bindings/python/tox.ini:24]: Is this checking for format or applying it ?

File: bindings/python/tox.ini (hunk 1 above)
Comment (qrayven) [reply bindings/python/tox.ini]: already fixed

File: bindings/python/tox.ini (hunk 18 above)
Comment (qrayven) [reply bindings/python/tox.ini:24]: Only checking. Do we want to modify?

File: bindings/python/iota_sdk/types/common.py (hunk 11 above)
Comment (qrayven) [reply bindings/python/iota_sdk/types/common.py]: removed

Comment (thibault-martinez) [reply bindings/python/iota_sdk/types/common.py]: Does it need a changelog entry?

File: bindings/python/tox.ini (hunk 18 above)
Comment (thibault-martinez) [reply bindings/python/tox.ini:24]: No no, checking is fine

File: bindings/python/iota_sdk/types/common.py (hunk 11 above)
Comment (qrayven) [reply bindings/python/iota_sdk/types/common.py]: removed for nodejs bindings too, changlog updated for both

**Review by Thoralf-M — DISMISSED** [review]

**Review by Thoralf-M — DISMISSED** [review]

File: bindings/nodejs/CHANGELOG.md [hunk 19]
```diff
@@ -29,11 +29,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+### Removed
+
+- `HD_WALLET_TYPE`, `HARDEN_MASK` constants;
+
 ### Fixed
 
 - Super class of `NftOutput` from `StateMetadataOutput` to `ImmutableFeaturesOutput`;
 - `Utils::parseBech32Address` now converts the string into a proper `Address`;
 
+
```
Comment (Thoralf-M) [inline bindings/nodejs/CHANGELOG.md]:
```suggestion
```

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**